BuildResidentialHPXML/measure.rb

```diff
@@ -1291,6 +1291,54 @@ def arguments(model)
     arg.setDefaultValue('76')
     args << arg
 
+    arg = OpenStudio::Measure::OSArgument::makeIntegerArgument('season_heating_begin_month', false)
+    arg.setDisplayName('Heating Season: Begin Month')
+    arg.setUnits('#')
+    arg.setDescription('This numeric field should contain the starting month number (1 = January, 2 = February, etc.) for the heating season.')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument::makeIntegerArgument('season_heating_begin_day_of_month', false)
```

Comment:
Since all of these arguments are optional.... what happens if they provide one of these 4 inputs but not the others?

Reply:
Then it defaults to year-round: https://github.com/NREL/OpenStudio-HPXML/blob/build-res-hpxml-v3-seasons/HPXMLtoOpenStudio/resources/hpxml_defaults.rb#L763-L783.

The same kind of logic happens for daylight saving time inputs: https://github.com/NREL/OpenStudio-HPXML/blob/build-res-hpxml-v3-seasons/HPXMLtoOpenStudio/resources/hpxml_defaults.rb#L99-L119.

Reply:
No, I think it will generate a schematron validation error since all 4 inputs are required when the HeatingSeason (or CoolingSeason) element is provided.

Reply:
And even if it defaulted to year-round, that would be confusing to enter values that are ignored.

Reply:
Tests added for "invalid" (incomplete) seasons.